Doc/library/asyncio-eventloop.rst

```diff
@@ -188,19 +188,21 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`ThreadPoolExecutor`. After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
```

Comment:
```suggestion
   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
   After calling this method,
   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
```

Fix broken reference to `ThreadPoolExecutor` (while still displaying it as such)